DataFormats/L1TGlobal/src/GlobalObject.cc

```diff
@@ -0,0 +1,154 @@
+/**
+ * \class GlobalObject
+ *
+ *
+ * Description: define an enumeration of L1 GT objects.
+ *
+ * Implementation:
+ *    <TODO: enter implementation details>
+ *
+ * \author: Vasile Mihai Ghete - HEPHY Vienna
+ *
+ *
+ */
+
+// this class header
+#include "DataFormats/L1TGlobal/interface/GlobalObject.h"
+
+// system include files
+#include <cstring>
+
+// user include files
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+using namespace l1t;
+
+
+l1t::GlobalObject l1TGtObjectStringToEnum(const std::string& label) {
+
+     
+    
+    static const l1t::L1TGtObjectStringToEnum l1TGtObjectStringToEnumMap[] = {
+            {"Mu",  gtMu},
+            {"EG",  gtEG},
+	    {"Tau", gtTau},
+            {"Jet", gtJet},
+            {"ETM", gtETM},
+            {"ETT", gtETT},
+            {"HTT", gtHTT},
+            {"HTM", gtHTM},
+	    {"ETM2", gtETM2},
+	    {"MinBias", gtMinBias},
+            {"External", gtExternal},
+            {"ObjNull", ObjNull},
+            {0, (GlobalObject) - 1}
+    };
+
+    l1t::GlobalObject value = (GlobalObject) - 1;
+
+    bool found = false;
+    for (int i = 0; l1TGtObjectStringToEnumMap[i].label && (!found); ++i)
```

Comment:
much cleaner to replace the found statements by a continue instead the if statement